src/styles/main.css

```diff
@@ -7,47 +7,294 @@
+  list-style: none;
+  text-decoration: none;
   color: #fff;
 }
 
-.links {
+.strng {
+  color: red;
+}
+
+/* Display item to the home page */
+.grid-cards-container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  margin-top: 6em;
+  gap: 30px;
+  padding: 50px;
+}
+
+.item {
   display: flex;
-  gap: 15px;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 0.6rem 0;
+  transition: 500ms ease-in;
+  border: 2px solid red;
+  background-color: #000;
 }
 
-.links li {
-  list-style-type: none;
-  font-size: 25px;
+.welcome {
+  text-align: center;
+  font-size: 2rem;
+  margin-top: 30px;
 }
 
-.links li:hover {
-  border-bottom: 2px solid #fff;
-  color: #000;
+.item:hover {
+  transform: scale(1.05);
 }
 
-.logo h1 {
-  border: 2px solid #fff;
-  border-radius: 1rem;
-  padding: 12px;
+.show-image {
+  width: 210px;
+  height: 295px;
 }
 
-.footer {
-  background-color: rgb(4, 4, 26);
+.card-title {
+  font-family: var(--main-font);
+  font-size: 1.2em;
   color: #fff;
+}
+
+.movie-title {
+  text-align: center;
+  width: 200px;
+}
+
+.comment-btn,
+.reserve-btn {
+  border: 2px solid;
+  padding: 0.4em;
+  font-family: var(--main-font);
+  font-weight: bold;
+  color: #fff;
+  cursor: pointer;
+  background-color: transparent;
+  border-radius: 8px;
+  margin-bottom: 20px;
+}
+
+.comment-btn:hover {
+  background-color: red;
+  color: #fff;
+}
+
+.comment-and-like {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+}
+
+.p-like {
+  color: #fff;
+  margin-top: -25px;
+}
+
+.heart {
+  width: 100px;
+  height: 100px;
+  background:
+    url("https://cssanimation.rocks/images/posts/steps/heart.png")
+    no-repeat;
+  background-position: 0 0;
+  cursor: pointer;
+  transition: background-position 1s steps(28);
+  transition-duration: 0s;
+  margin-bottom: 0;
+  margin-top: 0;
+}
+
+.heart.is-active {
+  transition-duration: 1s;
+  background-position: -2800px 0;
+}
+
+.loading {
+  width: 160px;
+  height: 160px;
+  border-radius: 150px;
+  border: 15px aquamarine;
+  border-top-color: rgba(0, 0, 0, 0.3);
+  box-sizing: border-box;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  margin-top: -80px;
+  margin-left: -80px;
+  animation: loading 1.2s linear infinite;
+  z-index: 999999999;
+}
+
+@keyframes loading {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.mask {
+  background: rgba(0, 0, 0, 0.918);
+  width: 100%;
+  height: 100vh;
   position: absolute;
+  z-index: 99999999;
+}
+
+.load-more {
+  padding: 8px;
+  background-color: aquamarine;
+  margin: 5px auto;
+  width: 40%;
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  border-radius: 3px;
+  border: none;
+  cursor: pointer;
+}
+
+/* //comment popup// */
+.movie-popup {
+  display: none;
+  background: #fff;
+  position: fixed;
+  z-index: 9999;
+  top: 0;
   bottom: 0;
+  right: 0;
+  left: 0;
+  overflow: auto;
+}
```

Comment:
# Optional Changes 

- [ ] I can see the popup modal is showing its contents as requested but it would be nice if you can reduce its width and height to not take the full screen on both the height and width. As it's it just feels like it's a new page, when it is supposed to be a modal.